src/helics/application_api/FilterOperations.cpp

```diff
@@ -273,13 +350,45 @@ void RerouteFilterOperation::setString(std::string_view property, std::string_vi
         }
         catch (const std::regex_error& re) {
             std::cerr << "filter expression is not a valid Regular expression " << re.what()
-                      << std::endl;
+                      << '\n';
```

Comment:
Is this to avoid flushing `cerr`?

Reply:
I think cerr is flushed anyway, so I don't think the flushing matters here.